src/com_tjnotifications/admin/models/notifications.php

```diff
@@ -253,17 +255,41 @@ public function getItems()
 						$providerConfigs['from_email'] = $json->from_email;
 					}
 
-					$providerConfigs['subject'] = $tConfig->subject;
-					$providerConfigs['body'] = $tConfig->body;
-					$providerConfigs['is_override'] = $tConfig->is_override;
+					if ($tConfig->provider == "email" && $tConfig->language != "*")
+	 				{
+	 					$emailLanguage[] = $tConfig->language;
+	 				}
+
+					if ($tConfig->provider == "sms" && $tConfig->language != "*")
+					{
+						$smsLanguage[] = $tConfig->language;
+					}
+					
+					$providerConfigs['subject']          = $tConfig->subject;
+					$providerConfigs['body']             = $tConfig->body;
+					$providerConfigs['is_override']      = $tConfig->is_override;
 					$providerConfigs['replacement_tags'] = $tConfig->replacement_tags;
-					$provider = $tConfig->provider;
+					$provider                            = $tConfig->provider;
 
 					$item->$provider = $providerConfigs;
 				}
+
+				$item->smsLanguages   = $smsLanguage;
+				$item->emailLanguages = $emailLanguage;
 			}
 		}
-
 		return $items;
 	}
+
+	/**
+	 * Get a list of the current content languages
+	 *
+	 * @return  array
+	 *
+	 * @since   4.0.0
+	 */
+	public function getLanguages()
+	{
+		return LanguageHelper::getContentLanguages(array(0,1));
```

Comment:
An error has occurred.
0 Class 'LanguageHelper' not found
`administrator/components/com_tjnotifications/models/notifications.php:293`